distributed/http/scheduler/prometheus/core.py

```diff
@@ -62,6 +62,16 @@ def collect(self):
             suspicious_tasks.add_metric([tp.name], tp.suspicious)
         yield suspicious_tasks
 
+        time_spent_tasks = CounterMetricFamily(
+            self.build_name("tasks_time_spent"),
+            "Total amount of time spent in each prefix",
+            labels=["task_prefix_name"],
+        )
+
+        for tp in self.server.task_prefixes.values():
+            time_spent_tasks.add_metric([tp.name], sum(tp.all_durations.values()))
```

Comment:
I propose breaking this out a bit.  I think that we'll want both compute and transfer separately.   The others tend not to be that important I don't think.

Reply:
What values are possible for StartStop action? I can see "compute" and "disk-write", but it's typed as any str